session_security/static/session_security/script.js

```diff
@@ -49,7 +50,12 @@ yourlabs.SessionSecurity.prototype = {
     // seconds.
     expire: function() {
         this.expired = true;
-        window.location.reload();
+        if (this.returnToUrl) {
```

Comment:
shouldn't this be !== undefined ?

Sorry for the late comment, i just thought i'd re-read before releasing this ^^